src/devices/Bmx280/BmxBase.cs

```diff
@@ -6,44 +6,37 @@
 //Formulas and code examples can also be found in the datasheet http://www.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf
 
 using System;
+using System.Buffers.Binary;
 using System.Device.I2c;
 using System.Threading.Tasks;
-using System.Buffers.Binary;
 
-namespace Iot.Device.Bmp280
+namespace Iot.Device.Bmx280
 {
-    public class Bmp280 : IDisposable
+    public class BmxBase : IDisposable
     {
-        private const byte Signature = 0x58;
+        internal I2cDevice _i2cDevice;
+        internal byte _signature;
```

Comment:
readonly?

Reply:
if I do understand correctly, readonly won't work as it won't be assigned in the base controller but in the child.